cli/src/main.rs

```diff
@@ -62,7 +66,7 @@ pub async fn main() {
   let anycloud_agz = base64::encode(include_bytes!("../alan/anycloud.agz"));
   let app = App::new(crate_name!())
     .version(crate_version!())
-    .about("AnyCloud is a Lambda alternative that works with multiple cloud provider.")
+    .about(concat!("Elastically scale webservers in any cloud provider.\nCurrent alan version: ", alan_version!()))
```

Comment:
nit: we can access the about cargo description as an environment variable so we only track it in one place https://stackoverflow.com/questions/27840394/how-can-a-rust-program-access-metadata-from-its-cargo-package